pom.xml

```diff
@@ -343,7 +345,7 @@
         <org.json.version>20240303</org.json.version>
 
         <!--temporary solr download url> -->
-        <solr.download.url>https://archive.apache.org/dist</solr.download.url>
+        <solr.download.url>https://dlcdn.apache.org</solr.download.url>
```

Comment:
This is the url you get from downloading solr manually, it is much faster than the archive url.